### ISSUE TITLE
Inform user when a tab is closed

### ISSUE THREAD

**Issue by ssorallen** [tabwrangler/tabwrangler#273]
A request from [Chris Merchant](https://crowdin.com/profile/chris.merchant.1914) via Crowdin:

Wondering if an optional pop up (static or temporary - user's choice) could be added to alert the user that tabs have just been archived? 

Power users like myself often use so many addons it can be easy to forget what's going on, suddenly notice tabs are missing and totally forget why they are gone. If this could be added and I could set it up to be static (others may prefer it to stick for their own set time period).

**Comment by gabefair** [comment]
Yes, this would be a nice enhancement. I would like to also ask for an option in the settings to both enable this feature and to also not show any notifications when tabs are closed. As I find the notifications distracting sometimes depending on my work.
